rasa_core/training/interactive.py

```diff
@@ -686,6 +686,15 @@ def _collect_messages(evts: List[Dict[Text, Any]]) -> List[Message]:
     for evt in evts:
         if evt.get("event") == UserUttered.type_name:
             data = evt.get("parse_data")
+            logger.info("Those are the collected entities:" + str(
+                data["entities"]))
+            for entity in data["entities"]:
+                if "extractor" in entity:
+                    # only inlcude entities which are extracted by ner_crf
+                    # or manually annotated (therefore lack entity["extractor"])
+                    if entity["extractor"] != 'CRFEntityExtractor':
```

Comment:
this should rather use `CRFEntityExtractor.__name__` to avoid a soft reference that can not be tracked by e.g. IDEs

Reply:
Why do we only include entities from the CRF, what if the user wrote his own entity component?

Reply:
See issue #1725. If the user wrote his own entity component the easiest solution would be if they just do not add an anything for `entity["extractor"]` depending on if they want it to be included in the training stories or not. Or we have to filter out the extractors we know never should be included here and if the user writes a new component it will by default be included in the stories.

Reply:
let's do it the other way around and test for the ones that we should not include as that is the way we do it during the NLU model evaluation where we only evaluate the processors that can be trained